pulp_puppet_plugins/pulp_puppet/plugins/importers/directory.py

```diff
@@ -262,8 +262,8 @@ def _import_modules(self, module_paths):
             _logger.debug(IMPORT_MODULE, dict(mod=module_path))
 
             module.set_storage_path(os.path.basename(module_path))
```

Comment:
Did some poking: ``import_content`` does not modify ``_storage_path`` so it's not equivalent to do ``module.import_content(module_path, location=os.path.basename(module_path))``. The location keyword is there to support directory FileContentUnits, ``set_storage_path`` is there to support single file units that need the ``_storage_path`` to point to the single file itself rather than the directory that holds it.